index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Survey Form</title>
+	<link rel="stylesheet" href="styles.css">
+</head>
+<body>
+	<center><section class="sec">
+		<div>
+			<h1 id="title">freeCodeCamp Survey Form</h1>
+			<p id="description">Thank you for taking the time to help us improve the platform</p>
```

Comment:
Please use the class instead of using id. Refer to **HTML & CSS best practices** where it says "_Always use the class attribute for multiple elements (do not use ID selector for multiple elements)._"

Reply:
-  [ ] Clear and polite comment, the reason for the change is given. 👍🏾 

- [OPTIONAL] Instead of bolding **HTML & CSS best practices**, I would suggest you make it a link that leads to the best practices